internal/epoch/epoch_manager.go

```diff
@@ -999,16 +1019,23 @@ func rangeCheckpointBlobPrefix(epoch1, epoch2 int) blob.ID {
 	return blob.ID(fmt.Sprintf("%v%v_%v_", RangeCheckpointIndexBlobPrefix, epoch1, epoch2))
 }
 
+func allowWritesOnIndexLoad() bool {
+	v := strings.ToLower(os.Getenv("KOPIA_ALLOW_WRITE_ON_INDEX_LOAD"))
```

Comment:
instead of that, can we pass this via Parameters ?

Reply:
@jkowalski I had a separate PR that wired this value, but decided against it.
As noted both in a previous PR and in the GH issue, the intent is for this to be a temporary "flag", so it should not make it into any (potentially) persistent data structure. We will remove this env var and epoch compaction code path from the read path, soon after enabling epoch compaction in maintenance. Does that make sense?

Reply:
makes sense. LGTM

I reviewed other PRs and I must commend you on all the research and refactoring work you did there.